### PR TITLE
Add domain_layer_location for Go

### PR DESCRIPTION
**Pull request by pongad**

### CODE REVIEW

**Comment by pongad** [comment]
@garrettjonesgoogle Not sure if you'd be interested in this?

File: google/logging/v2/logging_gapic.yaml [hunk 1]
```diff
@@ -6,6 +6,7 @@ language_settings:
     package_name: google.cloud.gapic.logging.v2
   go:
     package_name: cloud.google.com/go/logging/apiv2
+    domain_layer_location: cloud.google.com/go/logging
```
Comment (jba) [inline google/logging/v2/logging_gapic.yaml]: OK, not a dealbreaker, but in an ideal world we'd point to two packages, logging and logging/logadmin.

Just file as a bug or TODO, don't worry about it now.

Comment (garrettjonesgoogle) [inline google/logging/v2/logging_gapic.yaml]: @jba , what is logging/logadmin?

Comment (jba) [inline google/logging/v2/logging_gapic.yaml]: We split the handwritten logging client into two packages: logging, for writing logs, and logging/logadmin, for reading logs and the other stuff.

**Pull request merged**